src/util/getPageCategory.ts

```diff
@@ -5,6 +5,7 @@ const categories = [
 	['reference/', 'Reference'],
 	['guides/deploy/', 'Deploy Guides'],
 	['guides/integrations-guide/', 'Integration Guides'],
+	['reference/errors/', 'Reference'],
```

Comment:
I might suggest making this its own contained category. This is only for search purposes and I think there it could be helpful to separate errors from general reference material:
```suggestion
	['reference/errors/', 'Error Reference'],
```